x-pack/plugins/watcher/public/sections/watch_edit/components/threshold_watch_edit/threshold_watch_action_accordion.tsx

```diff
@@ -85,7 +85,7 @@ export const WatchActionsAccordion: React.FunctionComponent<Props> = ({
 
       return (
         <EuiAccordion
-          initialIsOpen={action.isNew}
+          initialIsOpen={action.isNew || hasErrors}
```

Comment:
If a user edits a threshold alert with a webhook action + basic auth set, the password will be redacted and will need to be reset.

Reply:
Would you mind leaving a comment in the code to explain this UX intention?